### PR TITLE
Supplying UTC TZ to date filter

### PR DESCRIPTION
**Pull request by 48klocs**
#1714 pointed out that reviews can appear to be submitted in the future.

I've verified with the API folks that the timestamps returned are in UTC.

### CODE REVIEW

**Comment by bhollis** [comment]
Hmm, can we correct the timestamp itself (i.e., parse it into the correct date when we load it)? I'm not 100% sure, but reading the documentation for the `date` filter it looks like it'll display the date in UTC, not parse the date as UTC.

**Comment by 48klocs** [comment]
Yeah, Angular's docs don't make the outcome super clear. I can double check on that later to verify if this is doing the trick or if coercing a TZ into the input is what's needed.

**Comment by delphiactual** [comment]
review.timestamp = review.timestamp + ' UTC';

https://stackoverflow.com/a/6525602

File: src/scripts/item-review/item-review.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <div class="community-review--who">
       <star-rating rating="review.rating" read-only></star-rating>
       <div ng-bind="review.reviewer.displayName" ng-class="{'community-review--who__special': review.isHighlighted}"></div>
-      <div ng-bind="review.timestamp | date:'mediumDate' : 'UTC'"></div>
+      <div ng-bind="review.timestamp | utcToLocal:'mediumDate' : 'UTC'"></div>
```
Comment (delphiactual) [inline src/scripts/item-review/item-review.html]: can remove `:'UTC'`?

Comment (48klocs) [inline src/scripts/item-review/item-review.html]: Oops, yeah.

**Comment by delphiactual** [comment]
@48klocs just changing this appears to resolve the issue completely 
`<div ng-bind="(review.timestamp + 'Z') | date"></div>`
since we know that mySQL will never have the Z or any other timezone appended

**Comment by 48klocs** [comment]
The time stamp is coming from the API. On the off chance that they start returning TZ in a future release (or the serializer starts including it), I'd rather not get bitten. 

Point taken though.

**Comment by delphiactual** [comment]
@48klocs it looks like you have a commit that is missing

**Comment by 48klocs** [comment]
I thought I'd fixed and pushed them.  I was clearly mistaken.

**Comment by bhollis** [comment]
Awesome, thanks!!

**Pull request merged**